### ISSUE TITLE
Staging Review finding: When clicking "edit" address, focus is not managed

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#116272]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tool 2
**Product:** Claim Status Tool
**Feature:** VA Letters and Documents

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** TBD (Accessibility)

### Description

When clicking "edit" using enter, focus shifts to the "Save" button in the revealed form. I would expect focus to move to the first focusable element within the form. Note: On subequent form opens (user clicks "Cancel" and then clicks "Edit" again) focus does move to the top of form as expected. 

<details><summary>Link, screenshot or steps to recreate</summary>

https://staging.va.gov/records/download-va-letters/letters/letter-page

To be added. 

</details>

## Recommended action

Move focus to the first focusable of the revealed form. 

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
